doc/tutorials/arrays.md

```diff
@@ -72,14 +72,14 @@ kdb set user:/tests/sequence/#3 'Fourth Element'
 
 . As you can see above arrays can contain “empty fields”: The key `user:/tests/sequence/#2` is missing.
 
-For array elements with an index larger than `9` we must add **underscores** (`_`) to the basename, so we do not destroy the alphabetic order of the array. For example, to add an eleventh element to our array we use the following command:
+While regular keys are sorted alphabetically, array elements are sorted by natural ordering:
```

Comment:
I find this sentence hard to understand. How exactly are these array elements sorted by natural ordering? Does this affect the user in any way? The output of `kdb` is always alphabetical, no?

Reply:
The output stays the same, but for the input, Electra will automatically prepend the `_` to keep the natural ordering of the array equal to alphabetical ordering.

Reply:
I just realized, that my comment is probably a better explanation of what happens here than, what I've put in the documentation. Will update accordingly.